c2corg_ui/static/js/addassociationdirective.js

```diff
@@ -55,7 +56,6 @@ app.AddAssociationController = function(appApi, $attrs) {
   this.api_ = appApi;
 
 
-  this.dataset = $attrs['dataset'];
```

Comment:
Don't you need to declare ``this.dataset``, as you did for ``parentId`` and ``addedDocuments``?